app/graphql/types/profile_stats/amount_consumed.rb

```diff
@@ -0,0 +1,35 @@
+class Types::ProfileStats::AmountConsumed < Types::BaseObject
+  description 'Generic Amount Consumed based on Media'
+
+  field :time, Integer,
+    null: false,
+    description: 'Total time spent in minutes.'
+
+  def time
+    object['time']
+  end
+
+  field :media, Integer,
+    null: false,
+    description: 'Total amount of media.'
+
+  def media
+    object['media']
+  end
+
+  field :units, Integer,
+    null: false,
+    description: 'Total progress per media including reconsuming.'
```

Comment:
"per media" seems confusing.  How about "total progress of library including reconsuming"?